main.css

```diff
@@ -644,3 +670,665 @@
     margin-right: 3px;
   }
 }
+
+@media (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  body {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #172b4d;
+    width: 1440px;
+  }
```

Comment:
- [ ] Kindly make sure that the display looks fine and matching to the Figma design on width `768px` and wider, [as per requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=(Desktop%3A%20768px%20or%20wider)). (Again please keep in mind this [hint]())